packages/ui-components/src/helpers/common/validation.ts

```diff
@@ -77,3 +77,17 @@ export const stringContains =
       return match.test(val) ? true : message
     }
   }
+
+export const isUrl: GenericValidateFunction<string> = (value) => {
+  if (VALID_HTTP_URL.test(value)) {
+    return true
+  }
+  return 'Invalid URL'
```

Comment:
Include the word `Value` in error messages, cause it will get replaced with the actual name/label of the input. So something like `Value is not a valid URL`. Same goes for the other function.